src/Integration/LocalServices/ProjectSystemHelper.cs

```diff
@@ -264,12 +271,12 @@ public static IEnumerable<Guid> GetAggregateProjectKinds(IVsHierarchy hierarchy)
 
         public static bool IsKnownTestProject(IVsHierarchy vsProject)
         {
```

Comment:
Following the only call chain:
called by --> ProjectSystem.IsTestProject --> ProjectSystemFilter.IsAccepted
IsAccepted already does a null check. So this change is safe.